### PR TITLE
Feature/morgan/css

### PR DESCRIPTION
**Pull request by Mlberry0205**
# Description
--Add tests to home_spec
--Add classNames to button in form
--Add className to h3 "Loading..."

## Type of change

- [ ] Bug fix (fixes an issue)
- [x] New feature (adds functionality)
- [x] Refactor

# How Has This Been Tested?

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new bugs
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Is your code D.R.Y?
- [ ] Does it follow SRP?

# Questions:
Are there any questions or concerns you want addressed in a code review?

# Next Steps:

### CODE REVIEW

File: cypress/e2e/home_spec.cy.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ describe('As a user, when I load the application, I can see a collection of pain
     })
```
Comment (alyciacan) [inline cypress/e2e/home_spec.cy.js:7]: Hey @Mlberry0205 , can you explain what the .then in line 5 does? I looked it up in the docs and am not understanding its role

File: cypress/e2e/home_spec.cy.js [hunk 2]
```diff
@@ -19,53 +19,91 @@ describe('As a user, when I load the application, I can see a collection of pain
...
-      .get('.images-container').should('exist')
-      .get(':nth-child(1) > .images').should('be.visible')
-      .get(':nth-child(2) > .images').should('be.visible')
+    .get('.images-container').should('exist')
+    .get(':nth-child(1) > .images').should('be.visible')
+    .get(':nth-child(2) > .images').should('be.visible')
   })
   
-  it('should not display details for an individual painting', () => {
+  it('should be able to click on a painting a view details about it', () => {
```
Comment (alyciacan) [inline cypress/e2e/home_spec.cy.js:61]: I did some poking around to see what was going on with the last test, the one you have skipped. I noticed that in the Cypress window, this test has the same /undefined url, so I think that this test would be failing too except that you don't actually have any assertions after the browser navigates to the individual page view (which never loads). I think we need to get some more help to figure out why we can't fetch an individual painting info.

**Review by alyciacan — COMMENTED** [review]
Hey Morgan, I am not going to merge this one because there are failing/skipped tests- I think we want to wait till we figure out how to fix them before merging, but we should be able to do that tomorrow!

File: cypress/e2e/home_spec.cy.js (hunk 1 above)
Comment (Mlberry0205) [reply cypress/e2e/home_spec.cy.js:7]: Hey Alycia, The .then() is was waiting for our data to return before displaying it.

File: cypress/e2e/home_spec.cy.js (hunk 2 above)
Comment (Mlberry0205) [reply cypress/e2e/home_spec.cy.js:61]: That fix worked! The test is now passing!

**Review by alyciacan — APPROVED** [review]
Looks good! Just the one skipped test but we'll work on that together today.

**Pull request merged**